### DISCUSSION TITLE
Shortcut icon not displaying

### DISCUSSION THREAD

**Discussion by coraphise** [valinet/ExplorerPatcher#2716] [Q&A] [answered]
Before I start, I will admit I didn't spend much time looking; some, but not much. If this is already answered, I'm sorry.

I have one application on my computer which I pin to my taskbar where the icon just shows as a white paper with the corner folded, instead of the application icon. I tracked down the shortcut in Windows Explorer, it has the correct icon when viewed there. I can change the icon, but it has no effect on the taskbar display.

Every other application I've pinned shows the correct icon.

Are there any suggestions to fix this behavior? I'm not sure if it's an ExplorerPatcher issue somehow, or a Windows issue.

Thanks in advance!

**Comment by pyrates999** [comment]
Uninstall EP, reboot, and see if this still occurs.

Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.

Please include any other software you have installed that modifies the UI of windows.

**Reply by coraphise** [reply]
Thanks for the suggestion, but... today the shortcut icon displays as expected. I have no explanation for it.

With all the issues I'm having lately I'm just going to accept this one minor annoyance self-resolve and call it a win while I try to figure other things out.

Thank you very much for getting back to me so fast.

**Comment by coraphise** [comment, marked as answer]
SELF RESOLVED 🤷

**Comment by tallman69** [comment]
I have had that icon behavior happen to me from time to time particularly when I create a shortcut from the list of applications in the start menu.  What I have found that helps prevent the blank page icon, along with other shortcut icon issues, is to make my shortcuts directly from the file that launches the application.  To do that I right click on the application Icon in the start menu and then click on `Open file location` and then make the shortcut from that file.

**Reply by coraphise** [reply]
Interesting; I wonder what causes that sort of behavior. The shortcut seems like it's valid no matter how I make it, as in it has the proper icon, so I wonder why creating the shortcut through other means would change display behavior.

What I find funny is that the icon went blank later, and has since gone back to properly displaying the application icon. It may have even happened before Explorer Patcher was installed... I don't recall.

In any case, it shows up properly and it's usable.
